### PR TITLE
Update User Guide

### LINKED ISSUE #124: Update UG for help and riskprofile command
As a financial advisor, I need an updated User Guide for the help and riskprofile command, so that I can refer to it when I face some issues.

### PR DESCRIPTION
**Pull request by Daphne789**
Add FAQ

Update User Guide for help, questionnaire, riskprofile, list, and exit command

Update the command summary in the User Guide to include Questionnaire, Risk Profile, and Exit

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
+
+#### Precise expected outcome on success:
+* Successful message: Opened help window
+* A pop-up window lists all the main commands and displays a link to FAPro's user guide
 
 ![help window](images/helpWindow.png)
 
+### Viewing risk assessment questionnaire: `questionnaire`
+
+Displays questions that are going to be used for generating client risk profile levels.
```
Comment (glenngnng) [inline docs/UserGuide.md]: I think more concise language can be used. What do you think of changing it to "displays questions that are used for ..."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
+
+#### Format: `questionnaire`
+
+![questionnaire format](images/questionnaireFormat.png)
+
+#### Precise expected outcome on success:
+* Successful message: Opened questionnaire window
```
Comment (glenngnng) [inline docs/UserGuide.md]: I think we standardised the success message format. It should be like:

Message shown to the user:

```
_message_
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
+* A pop-up window shows the risk assessment multiple choice questions, including the grading and risk profile categories criteria
+
+![questionnaire window](images/questionnaireWindow.png)
+
+### Adds risk profile level to a contact:`riskprofile`
+
+Generates the client risk profile level based on their response of risk assessment questionnaire provided in Questionnaire tab and adds it to their contact.
```
Comment (glenngnng) [inline docs/UserGuide.md]: Also, just another formatting issue. We are removing full stops :D Minor issue!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
+![riskprofile result](images/riskProfileResult.png)
+
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning:**
+Once the risk profile label is added to the contact, it cannot be removed
+</div>
+
+#### Precise expected outputs on failure:
+If the RESULT is empty
+* Error message:
```
Comment (glenngnng) [inline docs/UserGuide.md]: Something similar here for formatting of error message, it should be:

Error message shown to the user:

```
message
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
+This command can also be used to update the client risk profile
+</div>
+
+#### Acceptable values for each parameter:
+* INDEX: Only accept positive integers less than the size of the contacts displayed
+* RESULT: Valid result format, 8 comma-separated characters from 'a' - 'e' (e,b,a,c,b,b,a,e)
+
+#### Precise expected outcome on success:
+* Successful message: "Added risk profile to Person: X", where X are the details of the person edited
+* A risk profile label with a specified color will be added to the contact.
```
Comment (glenngnng) [inline docs/UserGuide.md]: Same thing here about the fullstops! :D

File: docs/UserGuide.md [hunk 6]
```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
+Invalid command format! 
+riskprofile: Adds the risk profile of the person identified by the index number used in the last person listing. 
+Existing risk profile will be overwritten by the input.
+Parameters: INDEX (must be a positive integer) res/[RESULT]
+Example: riskprofile 1 res/a,e,b,d,c,a,d,e
+```
+![invalid risk profile 1](images/invalidRiskProfile1.png)
+
+If the RESULT is not separated by commas or not in the range of 'a' - 'e'
+* Error message:
```
Comment (glenngnng) [inline docs/UserGuide.md]: Same thing about the formatting of error message! Small thing!

File: docs/UserGuide.md [hunk 7]
```diff
@@ -281,12 +358,16 @@ Precise expected outputs on failure:
...
-Precise command format: `list`
+#### Format: 
+* `list`
 
-Precise expected outputs on success:
-A list of all contacts with their details will be shown.
+![list format](images/listFormat.png)
+
+#### Precise expected outputs on success:
+* Successful message: Listed all persons
```
Comment (glenngnng) [inline docs/UserGuide.md]: Same thing here about formatting success message!

**Review by glenngnng — CHANGES_REQUESTED** [review]
Minor formatting issues, otherwise LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (Daphne789) [reply docs/UserGuide.md]: I have changed it. Thank you for the suggestion!

File: docs/UserGuide.md (hunk 3 above)
Comment (Daphne789) [reply docs/UserGuide.md]: Fixed!

File: docs/UserGuide.md (hunk 4 above)
Comment (Daphne789) [reply docs/UserGuide.md]: Thank you!

File: docs/UserGuide.md (hunk 5 above)
Comment (Daphne789) [reply docs/UserGuide.md]: Fixed!

File: docs/UserGuide.md (hunk 6 above)
Comment (Daphne789) [reply docs/UserGuide.md]: Solved!
Thank you.

File: docs/UserGuide.md (hunk 7 above)
Comment (Daphne789) [reply docs/UserGuide.md]: Solved!
Thank you.

**Review by glenngnng — APPROVED** [review]
LGTM!

**Pull request merged**